nano/core_test/block_store.cpp

```diff
@@ -1400,6 +1395,42 @@ TEST (mdb_block_store, upgrade_v21_v22)
 	// Testing the upgrade code worked
 	check_correct_state ();
 }
+
+TEST (mdb_block_store, upgrade_v22_v23)
+{
+	if (nano::rocksdb_config::using_rocksdb_in_tests ())
+	{
+		// Don't test this in rocksdb mode
```

Comment:
This comment is not very useful. The fact that we are skipping this test in rocked mode is obvious from the code. What I would really like to know is why we are skipping this test in RocksDB mode.